src/plugins/intel_gpu/src/plugin/transformations/convert_fc_to_compressed.cpp

```diff
@@ -44,7 +44,10 @@ ConvertFullyConnectedToFullyConnectedCompressed::ConvertFullyConnectedToFullyCon
     auto convert_m = wrap_type<ov::op::v0::Convert>({weights_m});
 
     auto sub_const_m = wrap_type<ov::op::v0::Constant>(consumers_count(1));
-    auto subtract_m = wrap_type<ov::op::v1::Subtract>({convert_m, sub_const_m});
+    auto sub_convert_const_m = wrap_type<ov::op::v0::Convert>({sub_const_m});
+    auto sub_with_convert_m = wrap_type<ov::op::v1::Subtract>({convert_m, sub_convert_const_m});
+    auto sub_no_convert_m = wrap_type<ov::op::v1::Subtract>({convert_m, sub_const_m});
```

Comment:
Shouldn't first arg be `weights_m` here?

Reply:
Hmm, if there is a convert on zp branch, subtract's (`sub_no_convert_m`) first input is always `convert_m`:
```
       weights_m   sub_const_m    
          |        /                         
       convert_m  sub_convert_const_m 
           \     /                               
        sub_with_convert_m      scale             
                        \      /
                        Multiply
```
As well as for `sub_no_convert_m` - `convert_m` is a first arg, isn't?
```
       weights_m   
          |                                 
       convert_m  sub_const_m    
           \     /                               
         sub_no_convert_m   scale             
                       \      /
                       Multiply
    
```